companies/iitr-cert-com.json

```diff
@@ -0,0 +1,30 @@
+{
+    "slug": "iitr-cert-com",
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "IITR Cert GmbH",
+    "address": "Eschenrieder Str. 62c\n82194 Gröbenzell\nDeutschland",
+    "phone": "+49 89 18917360",
+    "email": "email@iitr.de",
+    "web": "https://www.iitr-cert.com",
+    "sources": [
+        "https://www.iitr-cert.com/datenschutz/",
+        "https://www.iitr-cert.com/impressum/"
+    ],
+    "required-elements": [
+        {
+            "desc": "Name",
+            "type": "name"
+        },
+        {
+            "desc": "Address",
+            "type": "address"
+        },
+        {
+            "desc": "Email",
+            "type": "input"
+        }
+    ],
+    "request-language": "en"
```

Comment:
Any reason to choose English here?

Reply:
That's another of those fields that are more difficult to define. I only set the `request-language` very rarely. Usually, we want the default behaviour (i.e. falling back to the user's interface language).

If a company offers their services in more than one language, I don't set the rl. If they offer their services in the language corresponding to the relevant country (i.e. a company offering its services to German customers in German), I don't set it either.  
I really only set it if a company offers its services in multiple countries but only in one language (this is often the case for US startups, for example). Another possible case are German government agencies.

One more special case: We have some special templates for certain German government agencies (like the Bundespolizei). These templates are only available in German, so the rl has to be set here, as well.